### ISSUE TITLE
Ethos 1.5.18 telemetry unstable communication with sensor MLVSS

### ISSUE THREAD

**Issue by Richrfl** [FrSkyRC/ETHOS-Feedback-Community#4689]
I have flown two planes today both with RXs Archer Plus SR-10+, both of which have been reset to Factory after FW upgrade to V.11. and in both cases, I received a "sensor lost" warning, periodically and the TX's screen showed last value read in red. The RX telemetry, like RSSI and VFR, were OK at all times, as well as the data from the GPS. The logs show data at all times, but it seems to me that while the telemetry is lost, it fills it up with the last reading. This did not happen with the previous versions of Ethos and the RXs.  The MLVSS sensor is loaded with the latest FW (2022-07-11).
I set the delay for sensor loss to 5sec and also tried to 10sec, but there were no changes.

**Comment by bsongis-frsky** [comment]
There is only one MLVSS in your setup?
What is its update frequency for telemetry?

**Comment by Richrfl** [comment]
Yes, there is only one. I guess this is the frequency you are referring to is the settings in the Special Functions >Write Logs>Write Interval. I have set this for let's call it Model 1 to the default 250ms and I had Model 2 set at 500ms. Both have the same problem. Also, on both models, I have a GPS.

**Comment by Wolepo** [comment]
I have now built a counter with an LS, a Var and a corresponding announcement for each sensor, including the calculating ones. I use an X10S Express and an Archer SR8Pro as transmitters. The telemetry runs with S.Port. There are 29 sensors. I have now realised that the sensors VFR, GPS, GSpd and the sensors of the speed sensor cause the most ‘sensor lost’ messages. I have set the delay time to 1s for all sensors except GAlt (2s) and the two temperature sensors of the speed sensor (3s). 
In the case of a sensor lost message that really comes from a sensor, this message is output first and then the corresponding name is given by the SF. However, there are also ‘sensor lost’ messages without a sensor. It is difficult to understand.
 Today, however, I have an Archer plus SR8+ with MLVS AVD, GPS AVD and Vario AVD and a Neuron 60 II connected to S.Port. There are constant sensor lost messages and even sensor conflicts. If I use FPORT, there is surprisingly no problem. All sensors have the latest firmware.

**Comment by bsongis** [comment]
@Wolepo please contact me on Discord with this setup ready! I will send you a firmware which will dump the telemetry on SD. Then we will understand where is the problem!

**Comment by Richrfl** [comment]
@bsongis Bert, did walepo provide you with the requested information? I would like to see this issue solved. Can I help in any way?

**Comment by Wolepo** [comment]
Yes, we're on it

**Comment by Richrfl** [comment]
Tku for the update.

**Comment by Richrfl** [comment]
One more item related to the telemetry. I set a TD SR10 and found out in the telemetry screen that a sensor named R9M power has its poll dot blinking and shows 25mW even with the RX OFF. Power cycling the TX does not make any change.

**Comment by Richrfl** [comment]
@Wolepo @bsongis . I checked the newly released V19, and the description does not mention the telemetry issue in this thread. Version 1.6.0 has a vague address to the Telemetry, but it is not clear if this issue is solved as well as issue #61.
With that said, I would like to ask you guys in which version can I expect for these two issues to be taken care of.
At this point, I am very tempted to fall back to previous versions as V18 added a lot of new features and enhancements to the cost of creating bigger and more difficult-to-solve issues than expected and as of today, seems to me that have not being solved.

**Comment by bsongis** [comment]
Honestly I am not sure that the issue is linked to Ethos. I will tell here after the investigation is finished.

**Comment by Richrfl** [comment]
Thank you Bert.

**Comment by Richrfl** [comment]
Thank you very much for the heads up Ben.

**Comment by Richrfl** [comment]
Thank you for the update Ben, I noticed we "skipped" 1.5.20 and I was wondering...
